android/src/main/java/vn/luongvo/kmm/survey/android/ui/common/AlertDialog.kt

```diff
@@ -0,0 +1,43 @@
+package vn.luongvo.kmm.survey.android.ui.common
+
+import androidx.compose.material.*
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color.Companion.White
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import vn.luongvo.kmm.survey.android.R
+import vn.luongvo.kmm.survey.android.ui.theme.Nero90
+
+@Composable
+fun AlertDialog(
+    message: String,
+    onConfirmButtonClick: () -> Unit,
+) {
+    AlertDialog(
+        text = { Text(text = message) },
+        backgroundColor = White,
+        confirmButton = {
+            Button(
+                onClick = { onConfirmButtonClick() },
+                colors = ButtonDefaults.buttonColors(
+                    backgroundColor = Nero90
+                ),
+            ) {
+                Text(
+                    text = stringResource(id = R.string.generic_ok),
+                    color = White
+                )
+            }
+        },
+        onDismissRequest = { onConfirmButtonClick() }
```

Comment:
@luongvo Hmm, it's weird, we invoke confirmButtonClick on Dismiss? 🤔 
I suggest adding more onDismissListener is reasonable. What do you think?

Reply:
@doannimble I think it depends on what's our logic with it. In my case, dismissing the dialog or clicking on the OK button would simply close it without extra action. I think we will need to separate these actions in case we have different logic between `dismissing` and `confirming`, in my logic -> no, they're the same 🤔

Reply:
@luongvo I got your point, it is okay if we define for the LoginScreen only. But we are defining the common case, I guess we don't like to edit it in case of reusing.

Reply:
@doannimble Alright, I see the issue between overlapping the events of confirm & dismiss here. I just made an improvement https://github.com/luongvo/kmm-survey/pull/54/commits/ac8e0e68becdd3e6ef536be2181425a74697861b, please check it out. The idea is having 2 separate events but the default of confirming is dismissing to close the dialog 🤗

Reply:
Nice, thanks @luongvo